src/components/Island.scss

```diff
@@ -6,4 +6,8 @@
   border-radius: var(--border-radius-m);
   padding: calc(var(--padding) * var(--space-factor));
   position: relative;
+
+  &.zen-mode {
```

Comment:
Any reason for this doing this? I'm not sure how it will contribute to this PR.

Reply:
This was requested in the OP issue, and I agree it looks better without the shadow.